tests/unit/test_read_config.py

```diff
@@ -238,12 +299,34 @@ def test_009_config_object(input_str, config_path):
         if env_var in config.keys():
             config.pop(env_var)
 
+    # parse the dates
+    config["end_date"] = process_date_string(config["end_date"])
+    config["start_date"] = process_date_string(config["start_date"])
+
     # load config object (performs all of the above steps)
-    wrf_config = load_wrf_config(config_path)
+    wrf_config = load_wrf_config(config_path_wrf_docker)
 
     assert config == asdict(wrf_config)
 
 
+# TODO: Duplicate test? An invalid config would be caught by the integration test as well.
+def test_011_valid_CMAQ_NCI_config_file(config_path_cmaq_nci):
+    try:
```

Comment:
You don't need the try/except block here because if an exception is raised it will cause the test to fail anyway